### ISSUE TITLE
QuizSubmission update_score_and_comments doesn't seem to work

### ISSUE THREAD

**Issue by breed** [ucfopen/canvasapi#601]
# Describe the bug

when i can update_score_and_comments(fudge_points=points) on a quiz submission, i get the error "missing required key :quiz_submissions" back. looking through the code it doesn't seem like the data being sent to the canvas server matches the required format: https://canvas.instructure.com/doc/api/quiz_submissions.html#method.quizzes/quiz_submissions_api.update

# To Reproduce

         for s in quiz.get_submissions():
             s.update_score_and_comments(fudge_points=9)

# Expected behavior

the fudge_points get set to 9

# Environment information

- Python version (`python --version`) Python 3.10.6
- CanvasAPI version (`pip show canvasapi`) Name: canvasapi Version: 3.0.0

# Additional context

**Comment by breed** [comment]
dang, i though i had tried that! thank you for the super quick response. that works thank you!

it seems a little off to have to do the wrapping in the quiz_submissions, since that is always needed, and to add the attempt, since that is in s (the QuizSubmission object). i've put up #602 to make this a little nicer while still preserving legacy behavior.

**Comment by bennettscience** [comment]
Yeah, there's a few places the API expects a list or dict without really being specific - the only way you can tell is based on how they note the parameters in the docs. It bit me in the but a lot when I first started.

I like the idea of wrapping for people as the default in the library...curious what @Thetwam will say.

Glad I could help 👍
